src/test/java/testresults/PostTestResultsPozMainLvlSubmitted.java

```diff
@@ -530,29 +530,27 @@ public void testResultsValueVehicleTypeValueOne() {
     }
 
 
-    @Ignore
+    @Ignore("Ignored until CVSB-6491 is solved")
     @Title("CVSB-417 - CVSB-949 - CVSB-1140 / CVSB-1573 - Consumer creates a new test results for the submitted/cancelled test - vehicleType hgv")
     @Test
     public void testResultsValueVehicleTypeValueTwo() {
 
         testResultsSteps.postTestResults(vehicleSubmittedData.setVin(generateRandomExcludingValues(21, vehicleSubmittedData.build().getVin()))
                 .setVrm(generateRandomExcludingValues(7, vehicleSubmittedData.build().getVrm()))
                 .setVehicleType("hgv").build());
-
         testResultsSteps.statusCodeShouldBe(201);
         testResultsSteps.validateData("Test records created");
         validateSavedData();
     }
 
-    @Ignore
+    @Ignore("Ignored until CVSB-6491 is solved")
```

Comment:
Suggest adding a comment or a subtask to 6491 to re-enable the tests as part of the solution ;)